lib/controllers/AssetController.ts

```diff
@@ -37,6 +45,10 @@ export class AssetController extends CRUDController {
             { verb: 'get', path: 'device-manager/:index/assets/_search' },
           ],
         },
+        importAssets: {
+          handler: this.importAssets.bind(this),
+          http: [{ verb: 'post', path: 'device-manager/assets/_import' }]
```

Comment:
You can add the `index` in the URL to follow the same naming convention as the other routes
```suggestion
          http: [{ verb: 'post', path: 'device-manager/:index/assets/_import' }]
```